lego/apps/users/models.py

```diff
@@ -552,24 +552,33 @@ class PhotoConsent(BasisModel):
     class Meta:
         unique_together = ("semester", "year", "domain", "user")
 
-    def get_consents(self, user):
+    def get_consents(self, user, *, time=None):
```

Comment:
The `*` param just means that all the parameters following are named, so its not possible to pass an unnamed argument for a named parameter. It just makes the code more safe, especially when refactoring.

I don't see why the name here matters really? Since its quite explanatory and is not necessarily connected to the serializer, but if you really want to I can change it.

Reply:
Cool, that makes sense. And the name is quite explanatory, indeed, but I would just like it to be perfect ..